common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -535,7 +510,7 @@ private static boolean isSchemaCompliant(final Class<?> clazz, final String[][]
         return isCompliant;
     }
 
-    private static boolean isAccountSchemaCompliant(@NonNull final Account account) {
+    boolean isAccountSchemaCompliant(@NonNull final Account account) {
```

Comment:
I am expecting the implementation of this method to change slightly because B2C isn't populating a `realm` at the moment.